api/routes/saved_jobs.py

```
@@ -0,0 +1,304 @@
+            # Just get saved job records
+            response = (
+                supabase.table("saved_jobs")
+                .select("*")
+                .eq("user_id", user_id)
+                .order("saved_at", desc=True)
+                .execute()
+            )
+
+        return response.data
+
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)